tests/linalg_test.py

```diff
@@ -217,6 +236,26 @@ def norm(x):
     self._CompileAndCheck(partial(np.linalg.eigh, UPLO=uplo), args_maker,
                           check_dtypes=True, rtol=1e-3)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_shape={}".format(
+           jtu.format_shape_dtype_string(shape, dtype)),
+       "shape": shape, "dtype": dtype, "rng": rng}
+      for shape in [(4, 4), (5, 5), (50, 50)]
+      for dtype in float_types + complex_types
+      for rng in [jtu.rand_default()]))
+  # TODO: enable when there is an eigendecomposition implementation
```

Comment:
There is a *symmetric* eigendecomposition implementation for both platforms, although the precision of the TPU one isn't great and only supports real inputs. The GPU one should be pretty good and supports everything CPU does.

I'd just enable this test everywhere, although to be really sure you'd need to test against the other platforms, which our Github presubmits don't do.